Tasks/AzureRmWebAppDeployment/xdtutility.ts

```diff
@@ -0,0 +1,48 @@
+import path = require('path');
+import tl = require('vsts-task-lib/task');
+import trm = require('vsts-task-lib/toolrunner');
+
+function applyXmlTransformation(sourceFile, transformFile) {
+	var cttPath = path.join(__dirname, 'ctt/ctt.exe');
+    var cttArgs = ' s:"' + sourceFile + '" t:"' + transformFile + '" d:"' + sourceFile + '" pw';
```

Comment:
Indicates 'preserve whitespaces' in the output xml. It is just used for indentation.